lib/ansible/modules/web_infrastructure/bugzilla_bug.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) 2017, Bhavik Bhavsar <9.bhavik@gmail.com>
+#
+# This file is part of Ansible
+#
+# Ansible is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# Ansible is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with Ansible.  If not, see <http://www.gnu.org/licenses/>.
+
+ANSIBLE_METADATA = {
+    'status': ['preview'],
+    'supported_by': 'community',
+    'metadata_version': '1.0'
+}
+
+DOCUMENTATION = '''
+---
+module: bugzilla_bug
+short_description: Fetch details about a bug from a given bugzilla server
+description:
+    - Details such as status, resolution, severity, etc.,
+      can be fetched for a particular I(bug_id).
+version_added: "2.4"
+author: Bhavik Bhavsar
+requirements:
+    - "python >= 2.6"
```

Comment:
Will remove this, I had added this previously since it was checking for python 2.4 version